packages/design-system/src/components/web-components/ds-accordion/ds-accordion-item.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-accordion-item';
+
+export default {
+  title: 'Web Components/AccordionItem',
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    'button-class-name': {
+      control: 'text',
+      description: 'Class to be applied to the header `<button>` of an accordion item.',
+    },
+    'content-class-name': {
+      control: 'text',
+      description: 'Class to be applied to the content `<div>` tag of an accordion item.',
+    },
+    'default-open': {
+      description: 'Set to "true" to apply a border to the accordion content.',
+      control: 'boolean',
+    },
+    heading: {
+      control: 'text',
+      description: 'Class to be applied to the outer `<div>` that contains all accordion items.',
+    },
+    'heading-level': {
+      description: 'Heading type to override default `<h2>`',
+      options: ['1', '2', '3', '4', '5', '6'],
+      control: { type: 'select' },
+    },
+  },
+  args: {
+    heading: 'Detailed information',
+    'heading-level': '2',
+  },
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component: `For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/accordion/).`,
+      },
+    },
+  },
+  decorators: [webComponentDecorator],
+};
+
+const Template = (args) => (
+  <ds-accordion-item {...args}>
+    This is some detailed information inside an accordion item.
```

Comment:
Something's up with this Story - not seeing a border on my local Storybook
<img width="681" alt="image" src="https://github.com/CMSgov/design-system/assets/5159392/6ba7977e-6761-4a42-9365-b018e5711d4a">

Reply:
Does AccordionItem need a Story? React doesn't have one.

Reply:
(Also, Accordion does have the correct borders, so my screenshot probably means nothing then)

Reply:
[See commit note](https://github.com/CMSgov/design-system/pull/3071/commits/e336ff749dec47434de8740157f17ca708df50c3) - I only included it because otherwise we wouldn't have a docs page for that one